lib/src/serializable.dart

```diff
@@ -0,0 +1,237 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+library serializable_module.src.serializable;
+
+@MirrorsUsed(metaTargets: 'serializable_module.src.serializable.Reflectable')
+import 'dart:mirrors';
+import 'dart:async';
+import 'dart:html' show window, CustomEvent;
+import 'dart:js' show JsObject, context;
+
+import 'package:logging/logging.dart';
+import 'package:w_common/w_common.dart' show JsonSerializable;
+
+import 'event.dart';
+import 'module.dart';
+
+// Any classes / methods that are going to be reflected must annotate with this
+class Reflectable {
+  const Reflectable();
```

Comment:
No you have to define your own class you annotate with. https://api.dartlang.org/dev/1.20.0-dev.9.0/dart-mirrors/MirrorsUsed/metaTargets.html